src/neo/SmartContract/InteropService.cs

```diff
@@ -15,6 +15,16 @@ static InteropService()
                 t.GetFields()[0].GetValue(null);
         }
 
+        public static long GetPrice(uint hash)
+        {
+            return methods[hash].GetPrice();
+        }
+
+        public static long GetPrice(uint hash, ApplicationEngine applicationEngine)
+        {
+            return methods[hash].GetPrice(applicationEngine);
```

Comment:
TryGet?

Reply:
I prefer to remove the method

`public static long GetPrice(uint hash, EvaluationStack stack)`

And use only this one

` public static long GetPrice(uint hash, ApplicationEngine applicationEngine)`